### ISSUE TITLE
Popover Tip Is Announced by Screen Readers

### ISSUE THREAD

**Issue by Jan-Eimertenbrink** [mittwald/flow#2499]
<img width="572" height="261" alt="Image" src="https://github.com/user-attachments/assets/7048e186-25ca-4a9d-a022-69efaa5289c2" />

The screen reader detects the visual tip of popovers, such as in ContextualHelp or the info button in the PasswordCreationField, as an image. This should not happen, since the tip is purely decorative and does not provide meaningful information. It should be hidden from the accessibility tree so it is not announced.
